Dependencies/xr/Source/ARKit/XR.mm

```diff
@@ -933,6 +1024,113 @@ void UpdatePlanes(std::vector<Frame::Plane::Identifier>& updatedPlanes, std::vec
             }
         }
 
+        /**
+         Updates existing meshes in place, gets the list of updated/created mesh IDs and removed mesh IDs.
+         */
+        void UpdateMeshes(std::vector<Frame::Mesh::Identifier>& updatedMeshes, std::vector<Frame::Mesh::Identifier>& deletedMeshes) {
+            if (![sessionDelegate TrySetMeshDetectorEnabled: true]) {
+                return;
+            }
+
+            if (@available(iOS 13.4, *)) {
```

Comment:
Shouldn't we not even get into this code if we aren't running at least iOS 13.4? Seems like this check is unnecessary here?

Reply:
I was thinking the same but Xcode views it as an error if I don't include @available whenever I reference ARMeshAnchor (or anything mesh related from ARKit) and the build fails without it